pkg/curatedpackages/curatedpackages.go

```diff
@@ -0,0 +1,51 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+
+	"github.com/aws/eks-anywhere-packages/pkg/artifacts"
+	"github.com/aws/eks-anywhere-packages/pkg/bundle"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+func NewRegistry(deps *dependencies.Dependencies, registryName, kubeVersion, username, password string) (BundleRegistry, error) {
+	if registryName != "" {
+		registry := NewCustomRegistry(
+			deps.Helm,
+			registryName,
+			username,
+			password,
+		)
+		return registry, nil
+	}
+	defaultRegistry := NewDefaultRegistry(
+		deps.ManifestReader,
+		kubeVersion,
+		version.Get(),
+	)
+	return defaultRegistry, nil
+}
+
+func CreateBundleManager(kubeVersion string) bundle.Manager {
+	major, minor, err := parseKubeVersion(kubeVersion)
+	if err != nil {
+		return nil
+	}
+	log := logr.Discard()
+	discovery := NewDiscovery(major, minor)
```

Comment:
I thought something was going to change with the Discovery struct no?

Reply:
Whoops don't know how I missed that. I'll include it in the next PR.